packages/pg/test/test-helper.js

```diff
@@ -125,6 +125,28 @@ assert.throws = function (offender) {
   assert.ok(false, 'Expected ' + offender + ' to throw exception')
 }
 
+/**
+ * Executes a function that should return a Promise that will reject.
+ *
+ * If the Promise resolves then fail.
+ * If the Promise does not reject within timeoutMillis then fail.
+ * If the Promise rejects then __return__ the Error
+ *
+ * NOTE: The error is returned so the caller can perform additional validation
+ */
+assert.rejects = function (offender, timeoutMillis) {
```

Comment:
Since `assert.rejects` is an existing builtin but behaves differently, I think this should be called something else. Or `assert.rejects(() => client.connect, {code: '28P01'})`.

Reply:
Ha! I knew I couldn't have come up with a name that good on my own.

I'll take a look tomorrow at which is available going back to v8.x and adjust accordingly. Thanks for the review!